spec/controllers/idv/in_person/ssn_controller_spec.rb

```diff
@@ -120,21 +71,8 @@
       expect(@analytics).to have_received(:track_event).with(analytics_name, analytics_args)
     end
 
-    it 'updates DocAuthLog ssn_view_count' do
-      doc_auth_log = DocAuthLog.create(user_id: user.id)
-
-      expect { get :show }.to(
-        change { doc_auth_log.reload.ssn_view_count }.from(0).to(1),
-      )
-    end
-
-    it 'adds a session id to flow session' do
-      get :show
-      expect(flow_session[:threatmetrix_session_id]).to_not eq(nil)
-    end
```

Comment:
Since this controller still has threatmetrix, this spec should probably still be included.

Reply:
good catch!